packages/ERTP/test/unitTests/mathHelpers/test-natMathHelpers.js

```diff
@@ -35,15 +35,18 @@ test('natMathHelpers', t => {
   t.deepEqual(getAmountMathKind(), MathKind.NAT, 'amountMathKind is nat');
 
   // make
-  t.deepEqual(make(4), { brand: mockBrand, value: 4 });
+  t.deepEqual(make(4), { brand: mockBrand, value: 4n });
   t.throws(
     () => make('abc'),
-    { instanceOf: RangeError, message: 'not a safe integer' },
+    {
+      instanceOf: TypeError,
+      message: 'abc is a string but must be a bigint or a number',
```

Comment:
is the exact message part of the API? Should the test fail if somebody adds a period at the end of the sentence?